engines/postgres.py

```diff
@@ -72,11 +72,10 @@ def insert_data_from_file(self, filename):
         self.get_cursor()
         ct = len([True for c in self.table.columns if c[1][0][:3] == "ct-"]) != 0
         if (([self.table.cleanup.function, self.table.delimiter,
-             self.table.header_rows] == [no_cleanup, ",", 1])
-            and not self.table.fixed_width
-            and not ct
-            and (not hasattr(self.table, "do_not_bulk_insert") or not self.table.do_not_bulk_insert)
-            ):
+                self.table.header_rows] == [no_cleanup, ",", 1]) and not
+                self.table.fixed_width and not
+            ct and (not hasattr(self.table, "do_not_bulk_insert") or not
+                    self.table.do_not_bulk_insert)):
```

Comment:
I find the variation in the indenting of this conditional confusing. I'd rather have the long one then have the last line split and lose track to the initial `and`. I also find having the `not`s at the end of the line confusing. Having the `and` there is fine, but I worry missing the not when reading the condition.